src/utils.ts

```diff
@@ -53,3 +61,29 @@ export async function toSettlementContract(
 ): Promise<Contract> {
   return new Contract(address, GPv2SettlementArtefact.abi, ethers.provider);
 }
+
+export class Signature {
+  signature: string;
+  signatureScheme: string;
+  constructor(signature: string, signatureScheme: string) {
+    this.signature = signature;
+    this.signatureScheme = signatureScheme;
+  }
```

Comment:
ubernit: We can probably make these fields read-only and let the compiler set them for us:

```suggestion
  constructor(
    public readonly signature: string,
    public readonly signatureScheme: string,
  ) {}
```